tracer.go

```diff
@@ -876,14 +878,22 @@ func (t *tracer) printAddressInfo(a AddressInfo) {
 	t.printf("}")
 }
 
-const maxBytes = 32
+var maxBytes = 32
+
+func init() {
+	if m := os.Getenv("TRACER_MAX_BYTES"); m != "" {
```

Comment:
How about having it be an option to `wasi.Trace`?

Environment variables automatically interpreted by libraries are always a bit of leaky abstractions that we're likely to regret down the road.

Reply:
See 42cc598